packages/runner-shared/src/event-manager.js

```diff
@@ -586,9 +586,13 @@ export const eventManager = {
     const crossOriginLogs = {}
 
     Cypress.primaryOriginCommunicator.on('log:added', (attrs) => {
+      // if the test is over and the user enters interactive snapshot mode, do not add cross origin logs to the test runner
+      if (Cypress.state('test')?.final) return
```

Comment:
I'm trying to check here if the spec is finished running. If so, we don't want to continue adding logs to the test to preserve the final snapshot state as well as prevent load/unload events from showing up in the command log (due to the possibility the `src` attribute is removed).

Reply:
Why do we want to hide the load/unload events?

Reply:
Only when the test is over and we `unload` the spec bridge after applying `about:blank` to get the same origin policy in order. This should only be a) after the spec is done and b) practically speaking for spec bridge unload events, Technically, load events I think can still happen but I can't think of a scenario where that happens.